azure/table_azure_key_vault_secret.go

```diff
@@ -201,6 +201,11 @@ func getKeyVaultSecret(ctx context.Context, d *plugin.QueryData, h *plugin.Hydra
 		splitID := strings.Split(*data.ID, "/")
 		vaultName = strings.Split(splitID[2], ".")[0]
 		name = splitID[4]
+
+		// Operation get is not allowed on a disabled secret
+		if !*data.Attributes.Enabled {
+			return nil, nil
+		}
```

Comment:
What will happen if someone has passed to make GET call, i.e. `select * from azure_key_vault_secret where name = 'secret123' and vault_name = 'vault123'`?